app/forms/refunds/original_case_details_form.rb

```diff
@@ -0,0 +1,73 @@
+module Refunds
+  class OriginalCaseDetailsForm < Form
+    PAYMENT_METHODS = ['unknown', 'card', 'cheque', 'cash'].freeze
+    COUNTRY_OF_CLAIMS = ['england_and_wales', 'scotland'].freeze
```

Comment:
Sometimes there are different countries, but because we can't handle them we change the postcode to SW55 9QT. So you might look out for those.

Reply:
The country of claim is purely a question that we ask the user - meaning where was it issued - and there are only 2 places where they get issued from.  Its purely for admin purposed to allow them to contact the correct office about the claim thats  all so should be no issue